components/events/AccordionSection.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <section
+    class="accordion__section"
+  >
+    <h2 class="copy__title">
+      {{ document.title }}
+    </h2>
```

Comment:
Nice to have one-liner:

```suggestion
    <h2 class="copy__title" v-text="document.title" />
```